src/plays/multiselect-component/styles.scss

```diff
@@ -0,0 +1,100 @@
+$select-component-width: 1000px;
+$select-component-font-size: 25px;
+$select-component-dropdown-max-height: 450px;
+
+.multi-select-container {
+  display: flex;
+  justify-content: center;
+}
+
+#select {
```

Comment:
do not use the reserved keywords as selectors. Use play specific name instead like '#multi-select-container-select`

Reply:
done